UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/FunctionalTests/EnsSubscriptionRequestInQueueTest.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Queues;
+using Azure.Storage.Queues.Models;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using UKHO.ExternalNotificationService.API.FunctionalTests.Helper;
+using UKHO.ExternalNotificationService.API.FunctionalTests.Model;
+
+namespace UKHO.ExternalNotificationService.API.FunctionalTests.FunctionalTests
+{
+    class EnsSubscriptionRequestInQueueTest
+    {
+        private EnsApiClient _ensApiClient { get; set; }
+        private TestConfiguration _testConfig { get; set; }
+        private D365Payload _d365Payload { get; set; }
+        private QueueClient _queue { get; set; }        
```

Comment:
Correct naming of properties please:
```suggestion
        private EnsApiClient EnsApiClient { get; set; }
        private TestConfiguration TestConfig { get; set; }
        private D365Payload D365Payload { get; set; }
        private QueueClient Queue { get; set; }        
```

Do you not get these suggestions in VS?:
![image](https://user-images.githubusercontent.com/91606761/154714504-f47ae649-7500-4fad-ba8b-f1017b5e6004.png)

Reply:
Thanks! done